functions.php

```diff
@@ -182,18 +182,34 @@ function thunderblog_output_featured_image_as_link() {
 	// load scripts
 	wp_enqueue_script( 'script', get_template_directory_uri() . '/assets/main.js', [], THUNDERBLOG_VERSION, true );
 	wp_enqueue_script( 'scriptmodule', get_template_directory_uri() . '/assets/share-on-mastodon_v1.2.0.min.js', [], '1.2.0', true );
-	add_filter('script_loader_tag', 'add_type_module' , 10, 3);
+	add_filter('script_loader_tag', 'thunderblog_add_type_module' , 10, 3);
 } );
 
+add_action( 'after_setup_theme', 'thunderblog_custom_logo_setup' );
+
 /**
  * Serve a script as module
  */
-function add_type_module($tag, $handle, $src) {
+function thunderblog_add_type_module($tag, $handle, $src) {
 	// if not your script, do nothing and return original $tag
 	if ( 'scriptmodule' !== $handle ) {
-			return $tag;
+		return $tag;
 	}
 	// change the script tag by adding type="module" and return it.
 	$tag = '<script type="module" src="' . esc_url( $src ) . '"></script>';
 	return $tag;
 }
+
+/**
+ * Enable header custom logo
+ */
+function thunderblog_custom_logo_setup() {
+	$defaults = array(
```

Comment:
You can use shorthand arrays (i.e. `[ ... ]`) here

Reply:
I would love to, but this is discouraged by WP CS 😅

Reply:
https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/#declaring-arrays

Reply:
I don't particularly buy into that reasoning considering other languages use `[ ... ]` but better to follow a style guide like that than mix it up. 

Thanks for the context!